crates/ide_completion/src/completions/record.rs

```diff
@@ -8,6 +8,8 @@ use crate::{
 };
 
 pub(crate) fn complete_record(acc: &mut Completions, ctx: &CompletionContext) -> Option<()> {
+    complete_record_literal(acc, ctx);
```

Comment:
Instead of calling this here, let's call this where `complete_record` is being called as these two are different completions.